### DISCUSSION TITLE
Creating native modules

### DISCUSSION THREAD

**Discussion by xhyrom** [mlua-rs/rlua#272] [Q&A] [answered]
Hello, i'm currently working on my small lua runtime for some things and i want to create custom native modules like `math` and not just set them into global object.

I don't know how to do it, because i don't saw any example and also any function that can do it.
I found other wrapper that know it - https://github.com/tomaka/hlua#creating-a-lua-module but its outdated.

Thanks ❤️

**Comment by xhyrom** [comment, marked as answer]
Found https://github.com/amethyst/rlua/issues/131#issuecomment-873545636 , sorry

**Reply by jugglerchris** [reply]
No problem, I hope it's working for you!

**Reply by xhyrom** [reply]
yep, it works
